### PR TITLE
[Tech] HR-modal window View User Information

### PR DESCRIPTION
**Pull request by kseniyaLine**

### CODE REVIEW

File: src/components/Employees/EmployeesTable/EmployeesTable.jsx [hunk 1]
```diff
@@ -17,7 +30,7 @@ export const EmployeesTable = (props) => {
   const filteredEmployees = useSelector((state) => state.employees.filteredEmployees);
 
   const filterEmployees = filteredEmployees ? filteredEmployees
-    .filter((el) => props.userName ? props.userName.toLowerCase() === el.name.toLowerCase() : el)
+      .filter((el) => props.userName ? props.userName.toLowerCase() === el.name.toLowerCase() : el)
```
Comment (mtalstykh-exadel) [inline src/components/Employees/EmployeesTable/EmployeesTable.jsx]: why did you add the space?

File: src/components/Employees/EmployeesTable/HRmodalWindowTestAssignment/HRmodalWindowTestAssignment.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import PropTypes from 'prop-types';
+import './HRmodalWindowTestAssignment.scss';
+import Button from '@material-ui/core/Button';
+import { Paper } from '@material-ui/core';
+import {englishLevel,priority} from './constans/constans';
+
+export const HRmodalWindowTestAssignment = ({name, handleClose}) => {
+
+  let itemKey = 0;
+  const modalBody = <>
```
Comment (tensenrey) [inline src/components/Employees/EmployeesTable/HRmodalWindowTestAssignment/HRmodalWindowTestAssignment.jsx]: what is it ?

File: src/components/Employees/EmployeesTable/EmployeesTable.jsx [hunk 3]
```diff
@@ -1,10 +1,23 @@
 import React, { useEffect, useState } from 'react';
-import { Button, Paper, Table, TableBody, TableCell, TableContainer, TableHead, TablePagination, TableRow } from '@material-ui/core';
+// import { Modal } from '@material-ui/core';
```
Comment (mtalstykh-exadel) [inline src/components/Employees/EmployeesTable/EmployeesTable.jsx]: remove comments

File: src/components/Employees/EmployeesTable/EmployeesTable.jsx [hunk 4]
```diff
@@ -1,10 +1,23 @@
...
+// import { useSelector } from 'react-redux';
```
Comment (mtalstykh-exadel) [inline src/components/Employees/EmployeesTable/EmployeesTable.jsx]: remove comments

**Review by mtalstykh-exadel — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]
